CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ##### Enhancements
 
+* Add support for `Configurable` types when dequeuing reusable views.
+[Will McGinty](https://github.com/willmcginty)
+[#53](https://github.com/BottleRocketStudios/iOS-UtiliKit/pull/58)
+
 ##### Bug Fixes
 
-* None.
+*
```

Comment:
Did you intentionally leave this as an empty bullet or should it say "None"?

Reply:
Was unintentional. Fixed.